core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/signup.jelly

```diff
@@ -48,19 +48,16 @@ THE SOFTWARE.
         <!-- mobile friendly layout -->
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <!-- css styling, will fallback to default implementation -->
-        <st:include it="${simpleDecorator}" page="simple-head.jelly" optional="true"/>
```

Comment:
why remove these optionals?

Reply:
Since the views are moved from `DefaultSimplePageDecorator` to `SimplePageDecorator` now, they can't be missing, so just clarifying that (although it doesn't change anything functionally)